src/__tests__/ComplaintCard.spec.jsx

```diff
@@ -11,7 +11,7 @@ describe('initial state', () => {
       company: 'ABC Corp',
       company_public_response: 'Lorem',
       company_response: 'Ipsum',
-      complaint_id: '99990909',
+      complaint_id: '<em99990909</em>',
```

Comment:
Is this formed correctly?

Reply:
![whoops](http://replygif.net/i/1149.gif)
Nope. And I'm declaring it explicitly below. Just forgot to remove this one. Thanks.